### PR TITLE
feat: migrate `model_performance` to T2D

### PR DESCRIPTION
**Pull request by HLasse**
- [x] I have battle-tested on Overtaci (RMAPPS1279)
- [ ] I have bumped the version as described in the [wiki](https://www.notion.so/Setting-up-Poetry-on-your-local-machine-76647d75e8fa4d9ba553cbc2a49946d9#dca1e0f2ce934a49acd1851aaf882d75)

## Notes for reviewers
Reviewers can skip X, but should pay attention to Y.

### CODE REVIEW

File: .aucs/aucs.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+run_id,auc
+2p2dnzhx,0.917
+2qsu8x6y,0.892
+1uuut992,0.917
```
Comment (MartinBernstorff) [inline .aucs/aucs.txt:4]: Add to .gitignore?

**Review by MartinBernstorff — APPROVED** [review]
Only a tiny comment

File: tests/test_model_performance.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+def test_binary_transform_from_dataframe_with_float(binary_float_df):
+    res = ModelPerformance.performance_metrics_from_df(
+        prediction_df=binary_float_df,
+        metadata_col_names="all",
+        prediction_col_name="scores",
+        label_col_name="label",
+    )
+
+    assert res[res["score_type"] == "acc"]["value"].values[0] == pytest.approx(0.666667)
```
Comment (MartinBernstorff) [inline tests/test_model_performance.py:110]: How does binary_transform work? Is there a threshold set for when to transform something to 1/0?

**Review by MartinBernstorff — APPROVED** [review]

File: tests/test_model_performance.py (hunk 2 above)
Comment (HLasse) [reply tests/test_model_performance.py]: Before: used 0.5 as threshold. Now: can be specified with `binary_threshold`

**Pull request merged**